packages/manager/modules/pci/src/projects/Project.class.js

```diff
@@ -1,26 +1,35 @@
+import { CLOUD_PROJECT_STATE, CLOUD_PROJECT_BILLING_STATE } from '../constants';
+
 export default class Project {
   /* @ngInject */
   constructor(project) {
     Object.assign(this, project);
   }
 
   isCreating() {
-    return this.status === 'creating';
+    return this.status === CLOUD_PROJECT_STATE.creating;
   }
 
   isDeleting() {
-    return this.status === 'deleting';
+    return this.status === CLOUD_PROJECT_STATE.deleting;
   }
 
   isSuspended() {
-    return this.status === 'suspended';
+    return this.status === CLOUD_PROJECT_STATE.suspended;
+  }
+
+  hasPendingDebt() {
+    return (
+      this.serviceInfo &&
+      this.serviceInfo.status === CLOUD_PROJECT_BILLING_STATE.PENDING_DEBT
+    );
```

Comment:
There is a `BillingService` class in `@ovh-ux/manager-models` that expose that kind of function based on the `serviceInfos`. It would be better to reuse than duplicate that logic

Reply:
This is no longer required. We get the info for all projects using the /services API with iceberg